pyproject.toml

```diff
@@ -13,18 +13,22 @@ exclude = [
 python = "^3.7.1"
 pandas = "^1.3.5"
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
 flake8 = "^5.0.4"
 isort = "^5.11.5"
 pytest = "^7.2.1"
+pylint = "*"
+black = "*"
+
 coverage = "^7.0"
 pytest-cov = "^4.0.0"
 pytest-dependency = "^0.5.1"
 pytest-tap = "^3.3"
 tabulate = "^0.9.0"
+bandit = "^1.7.5"
 
-[tool.poetry.group.dev.dependencies]
-pytest = "^7.3.1"
```

Comment:
The version you have removed is 7.3.1, a later version than that on line 19.

Reply:
Originally pytest was version 7.2.1 but after you worked on something it added another pytest dependency and it was version 7.3.1 so I am not sure if that was added accidentally or intentionally by you I think it was an accident from conversations we had a while ago so I kept the originally config of pytest version 7.2.1 and removed 7.3.1 as one of them had to removed =)

Reply:
Ok - I have a feeling I needed to update because of some functionality that was missing from the earlier version, but as I cant actually recall lets go with it and pick it up at a later point.